angular/src/components/send/add-edit.component.ts

```diff
@@ -106,8 +107,8 @@ export class AddEditComponent implements OnInit {
         this.disableHideEmail = await this.policyService.policyAppliesToUser(PolicyType.SendOptions,
             p => p.data.disableHideEmail);
 
-        this.canAccessPremium = await this.userService.canAccessPremium();
-        this.emailVerified = await this.userService.getEmailVerified();
+        this.canAccessPremium = this.activeAccountService.activeAccount.canAccessPremium;
+        this.emailVerified = await this.activeAccountService.get<boolean>(StorageKey.EmailVerified);
```

Comment:
I've tried to get away from ever calling activeAccount directly outside of the activeAccountService. If you'd like, a helper for emailVerified could work, but it would be pretty basic looking.